<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuaForm</class>
 <widget class="QWidget" name="LuaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="buttonOpen">
       <property name="toolTip">
        <string>Open script file</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonSave">
       <property name="toolTip">
        <string>Save script file</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonSaveAs">
       <property name="toolTip">
        <string>Save-As script file</string>
       </property>
       <property name="text">
        <string>Save As</string>
       </property>
       <property name="icon">
        <iconset theme="document-save-as">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonFont">
       <property name="toolTip">
        <string>Select display font</string>
       </property>
       <property name="text">
        <string>Font</string>
       </property>
       <property name="icon">
        <iconset theme="preferences-desktop-font">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonStart">
       <property name="toolTip">
        <string>Start script</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset theme="go-next">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonStop">
       <property name="toolTip">
        <string>Stop script</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset theme="process-stop">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="toolTip">
        <string>Clear Log</string>
       </property>
       <property name="text">
        <string>Clear Log</string>
       </property>
       <property name="icon">
        <iconset theme="edit-clear">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="CodeEditor" name="sourceEdit"/>
     <widget class="QPlainTextEdit" name="plainTextOutput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QTextEdit</extends>
   <header>CodeEditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>plainTextOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
